api/src/main/webapp/WEB-INF/app.yaml.template

```diff
@@ -0,0 +1,29 @@
+service: api
+runtime: java11
+env_variables:
+  GAE_PROFILER_MODE: "cpu,heap"
+  JAVA_USER_OPTS: "-Djava.util.logging.config.file=webapps/root/WEB-INF/logging.properties"
+  SPRING_DATASOURCE_DRIVER_CLASS_NAME: "${DB_DRIVER}"
```

Comment:
these are now being specified in both `application.properties` and here inthe env vars. Do we need both?

Reply:
Are all env vars can be passed into our app's docker? If yes, then we don't need them. 
Let me quickly try without those values